addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -251,9 +251,10 @@
                     <t t-if="availability_check" t-foreach="tickets" t-as="ticket">
                         <t t-foreach="range(1, ticket['quantity'] + 1)" t-as="att_counter" name="attendee_loop">
                             <t t-set="counter" t-value="counter + 1"/>
-                            <div class="modal-body">
+                            <input class="d-none" type="text" t-attf-name="#{counter}-event_ticket_id" t-attf-value="#{ticket['id']}"/>
+                            <div class="modal-body" t-if="event.specific_question_ids">
```

Comment:
@yagp-odoo 
Could you set the `t-if` before the `class` ? :slightly_smiling_face: 
It's better for readability
Thanks!